modules/ROOT/pages/troubleshoot-service-binding-issues.adoc

```diff
@@ -52,3 +53,37 @@ The status of binding displays as `ReferencesNonexistentInstance`.
 
 Delete the erroneous binding command and re-create it with the correct input.
 
+
+[[error-instance-not-ready]]
+== Error instance not ready
+
+When you create a binding, the referenced API or adapter must be ready. Otherwise, you receive the 'ErrorInstanceNotReady' error.
+
+== Causes
+
+This error can occur if:
+ 
+* You are trying to create an API binding and the API is not ready. This can be because the API provisioning failed or has not finished yet. 
+* You are trying to create an adapter binding and the adapter is not ready. This can be because the adapter provisioning failed or has not finished yet.
+
+
+=== Diagnose
+
+Diagnose this issue by running one of the following commands:
+
+* `$ asmctl api binding list`
+* `$ asmctl adapter binding list`
++
+The status of the binding displays as `ErrorInstanceNotReady`. Check the status of the referenced instance by running one of the following commands (depending if you are using api or adapter binding)
+
+* `$ asmctl api list`
+* `$ asmctl adapter list`
+
```

Comment:
Maybe you should need to add here the same comment about using `kubectl describe serviceinstances`:

```
In case there is no information above, you will have to run `kubectl describe serviceinstances <replace_with_api_name> -n <replace_with_namespace>`. In the `status` section of the resource, you will find the error message explaining the issue.
```

Reply:
Same as above.